wurst/systems/spells/gatherer/Radars.wurst

```diff
@@ -0,0 +1,339 @@
+    ..setButtonPositionNormalY(0)
+    ..setButtonPositionResearchX(2)
+    ..setButtonPositionResearchY(0)
+    ..setIconNormal(SPELL_ICON)
+    ..setIconResearch(SPELL_ICON)
+    ..setIconTurnOff(SPELL_ICON)
+    ..setHeroAbility(false)
+    ..setItemAbility(false)
+    ..setLevels(3)
+    ..setHotkeyNormal("W")
+    ..setName("Find sticks")
+    ..setDisableOtherAbilities(1, false)
+    ..setTooltipNormal(1, TOOLTIP_NORM_FINDSTICK)
+    ..setTooltipNormalExtended(1, TOOLTIP_EXT_FINDSTICK)
+    ..presetBaseOrderID(lvl -> "defend")
+    ..setOrderStringActivate("channel")
+    ..setOrderStringUseTurnOn("channel")
+    ..presetOptions(lvl -> 1)
+    ..presetCooldown(lvl -> COOLDOWN)
+    ..presetManaCost(lvl -> MANACOST)
+    ..presetFollowThroughTime(lvl -> CASTTIME)
+
+@compiletime function CreateFindMushroom()
+    new AbilityDefinitionIllidanChannel(ABILITY_ID_FINDMUSHROOM)
+    ..setTargetType(1, 0)
+    ..setAnimationNames("spell,slam")
+    ..setArtCaster("")
+    ..setArtEffect("")
+    ..setArtTarget("")
+    ..setArtSpecial("")
+    ..setButtonPositionNormalX(2)
+    ..setButtonPositionNormalY(0)
+    ..setButtonPositionResearchX(2)
+    ..setButtonPositionResearchY(0)
+    ..setIconNormal(SPELL_ICON)
+    ..setIconResearch(SPELL_ICON)
+    ..setIconTurnOff(SPELL_ICON)
+    ..setHeroAbility(false)
+    ..setItemAbility(false)
+    ..setLevels(3)
+    ..setHotkeyNormal("E")
+    ..setName("Find mushrooms")
+    ..setDisableOtherAbilities(1, false)
+    ..setTooltipNormal(1, TOOLTIP_NORM_FINDMUSHROOM)
+    ..setTooltipNormalExtended(1, TOOLTIP_EXT_FINDMUSHROOM)
+    ..presetBaseOrderID(lvl -> "curseon")
+    ..setOrderStringActivate("channel")
+    ..setOrderStringUseTurnOn("channel")
+    ..presetOptions(lvl -> 1)
+    ..presetCooldown(lvl -> COOLDOWN)
+    ..presetManaCost(lvl -> MANACOST)
+    ..presetFollowThroughTime(lvl -> CASTTIME)
+
+@compiletime function CreateFindClayball()
+    new AbilityDefinitionIllidanChannel(ABILITY_ID_FINDCLAYBALL)
+    ..setTargetType(1, 0)
+    ..setAnimationNames("spell,slam")
+    ..setArtCaster("")
+    ..setArtEffect("")
+    ..setArtTarget("")
+    ..setArtSpecial("")
+    ..setButtonPositionNormalX(2)
+    ..setButtonPositionNormalY(0)
+    ..setButtonPositionResearchX(2)
+    ..setButtonPositionResearchY(0)
+    ..setIconNormal(SPELL_ICON)
+    ..setIconResearch(SPELL_ICON)
+    ..setIconTurnOff(SPELL_ICON)
+    ..setHeroAbility(false)
+    ..setItemAbility(false)
+    ..setLevels(3)
+    ..setHotkeyNormal("R")
+    ..setName("Find clay balls")
+    ..setDisableOtherAbilities(1, false)
+    ..setTooltipNormal(1, TOOLTIP_NORM_FINDCLAYBALL)
+    ..setTooltipNormalExtended(1, TOOLTIP_EXT_FINDCLAYBALL)
+    ..setOrderStringActivate("channel")
+    ..setOrderStringUseTurnOn("channel")
+    ..presetBaseOrderID(lvl -> "curseoff")
+    ..presetOptions(lvl -> 1)
+    ..presetCooldown(lvl -> COOLDOWN)
+    ..presetManaCost(lvl -> MANACOST)
+    ..presetFollowThroughTime(lvl -> CASTTIME)
```

Comment:
Can we create a class which extends `AbilityDefinitionIllidanChannel` that encapsulates as much of this logic as possible that is shared between abilities? Then we can reuse it for each radar manipulation.